client/src/config/app/locale/en/json/glossary.json

```diff
@@ -95,6 +95,7 @@
     "selection_one": "selection",
     "selection_other": "selections",
     "notes_title_case": "Notes",
+    "notes": "notes",
```

Comment:
Should we add plural/singular for these, just in case we need them for a count somewhere?